MaxText/layers.py

```diff
@@ -32,14 +32,19 @@
 from flax.linen import partitioning as nn_partitioning
 
 import numpy as np
+import math
+import max_logging
 
 import jax
 from jax import lax
 from jax import random
 from jax.ad_checkpoint import checkpoint_name
 import jax.numpy as jnp
 from jax.experimental.pallas.ops.tpu import flash_attention
-
+try:
+  from jax.experimental.pallas.ops import attention as pallas_attention
+except ImportError:
+  max_logging.log("WARNING: jax_triton not found, please `pip install jax-triton`")
```

Comment:
Really should be `please install jax-triton to get best performance on GPU`.

Reply:
Like the two points are (1) are they really supposed to install jax-triton via pip and (2) TPU users shouldn't worry about this.